### PR TITLE
Add Permission map UI

### PR DESCRIPTION
**Pull request by tmkasun**
In this PR

- Add role permission UI

image

image

Jul-25-2020 15-40-54

### CODE REVIEW

File: features/apimgt/org.wso2.carbon.apimgt.admin.feature/src/main/resources/admin/source/src/app/components/RolePermissions/TreeView/PermissionsSelector.jsx [hunk 1]
```diff
@@ -0,0 +1,129 @@
+import React, { useState, useEffect } from 'react';
```
Comment (tharikaGitHub) [inline features/apimgt/org.wso2.carbon.apimgt.admin.feature/src/main/resources/admin/source/src/app/components/RolePermissions/TreeView/PermissionsSelector.jsx:1]: Let's add the licence header to this file.

File: features/apimgt/org.wso2.carbon.apimgt.admin.feature/src/main/resources/admin/site/public/pages/README.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
+<!--
+ * Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (tharikaGitHub) [inline features/apimgt/org.wso2.carbon.apimgt.admin.feature/src/main/resources/admin/site/public/pages/README.md:2]: Let's change the year to 2020.

File: features/apimgt/org.wso2.carbon.apimgt.admin.feature/src/main/resources/admin/source/src/app/components/RolePermissions/TreeView/PermissionsSelector.jsx (hunk 1 above)
Comment (tmkasun) [reply features/apimgt/org.wso2.carbon.apimgt.admin.feature/src/main/resources/admin/source/src/app/components/RolePermissions/TreeView/PermissionsSelector.jsx:18]: fixed with https://github.com/wso2/carbon-apimgt/pull/9045/commits/b857d16f62df6be3152b273bd00333653436d8b0

File: features/apimgt/org.wso2.carbon.apimgt.admin.feature/src/main/resources/admin/site/public/pages/README.md (hunk 2 above)
Comment (tmkasun) [reply features/apimgt/org.wso2.carbon.apimgt.admin.feature/src/main/resources/admin/site/public/pages/README.md:2]: fixed in https://github.com/wso2/carbon-apimgt/pull/9045/commits/b857d16f62df6be3152b273bd00333653436d8b0

**Pull request merged**